docs/ext/manager_tmpl.j2

```diff
@@ -8,17 +8,23 @@
 {% if cls._create_attrs %}
 **Object Creation**
 {% if cls._create_attrs[0] %}
-Mandatory attributes:
+Mandatory attributes for object create:
 {% for item in cls._create_attrs[0] %}
 - ``{{ item }}``
 {% endfor %}
 {% endif %}
 {% if cls._create_attrs[1] %}
-Optional attributes:
+Optional attributes for object create:
 {% for item in cls._create_attrs[1] %}
 - ``{{ item }}``
 {% endfor %}
 {% endif %}
+{% if cls._create_attrs[2] %}
+Mutually exclusive attributes for object create:
+{% for item in cls._create_attrs[2] %}
```

Comment:
Seems like the template is quite outdated, since we could now use `.required`, `.optional`, and `.exclusive` here instead of using the `[0]`, `[1]`, `[2]` indices I think. Could you check if that works @walterrowe? So something like this in the entire template:

```suggestion
{% if cls._create_attrs.exclusive %}
Mutually exclusive attributes for object create:
{% for item in cls._create_attrs.exclusive %}
```

Reply:
Does the lint checker need fixing?

```
ERROR: InvocationError for command /home/runner/work/python-gitlab/python-gitlab/.tox/lint/bin/cz check --rev-range 65ecadc..HEAD (exited with code 14)
___________________________________ summary ____________________________________
ERROR:   cz: commands failed
```

Reply:
@walterrowe that's the commit message linter complaining about the commit message format - you'll find the guidelines for that in our contributing guide. John already fixed it for you but maybe for the next PR :)

I can see it's probably a bit cryptic if you run the whole `tox` suite as we have a lot of stuff running there and it gets buried in the checks.